src/client.ts

```diff
@@ -369,6 +467,29 @@ export class Client {
   // FILE MARKER - METHODS - PRIVATE ///////////////////////////////////////////
   //////////////////////////////////////////////////////////////////////////////
 
+  /**
+   * This method is used internally to calculate the element Viewport (Dimensions)
+   * @param selector
+   * @returns ViewPort object
+   */
+  private async getViewport(selector: string) {
+    const Values: DOMRect = JSON.parse(
+      //Had to do this to make it parse
+      "" +
+        await this.evaluatePage(
+          `JSON.stringify(document.querySelector('${selector}').getBoundingClientRect())`,
+        ),
```

Comment:
should be able to do
```
 const values = JSON.parse(
   await this.evaluatePage(
     `JSON.stringify(document.querySelector('span').getBoundingClientRect())`
   ) as string
 } 
```
I plan on adding generics at some point which would be nicer